src/ucs/sys/topo/base/topo.c

```diff
@@ -379,9 +422,11 @@ ucs_topo_sys_device_set_name(ucs_sys_device_t sys_dev, const char *name)
         return UCS_ERR_INVALID_PARAM;
     }
 
-    ucs_free(ucs_topo_global_ctx.devices[sys_dev].name);
-    ucs_topo_global_ctx.devices[sys_dev].name = ucs_strdup(name,
-                                                           "sys_dev_name");
+    if (override_name) {
+        ucs_free(ucs_topo_global_ctx.devices[sys_dev].name);
+        ucs_topo_global_ctx.devices[sys_dev].name = ucs_strdup(name,
+                                                               "sys_dev_name");
```

Comment:
line seems too long (pls check clang fmt)

Reply:
it's exactly 80 chars, clang does not warn about it.